src/index.ts

```diff
@@ -366,7 +367,6 @@ function focusKeydown(event: KeyboardEvent) {
   if (key !== 'ArrowRight' && key !== 'ArrowLeft' && key !== 'Home' && key !== 'End') return
   const toolbar = event.currentTarget
   if (!(toolbar instanceof HTMLElement)) return
-  if (toolbar.hasAttribute('data-no-focus')) return
```

Comment:
Does this line need to be removed? I see it prevents arrow keys from functioning when the toolbar has the attribute.